docs/design/base-image.md

```diff
@@ -1,14 +1,14 @@
 # The Base Image
 
-The ["base" image][base image] is a small-ish Docker image for running
+The ["base" image][base image] is a small-ish container image for running
```

Comment:
This particular image is built from a `Dockerfile`, I'm not sure how helpful this change is.
I think podman / cri-o / containerd users know that a "docker image" is not limited to Docker. :^)